app/src/main/java/com/nervousfish/nervousfish/activities/MainActivity.java

```diff
@@ -46,77 +51,94 @@
     @Override
     protected void onCreate(final Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
-        setContentView(R.layout.activity_main);
-        LOGGER.info("MainActivity created");
```

Comment:
Moved it to the end of the method, see [line 100](https://github.com/ericcornelissen/NervousFish/pull/100/files#diff-149538cc1f1f5e4370b6ae93d6489330R99)